.pre-commit-config.yaml

```diff
@@ -75,3 +75,24 @@ repos:
           - flake8-print
         args:
           - --per-file-ignores=tests/*.py:T201 # prints are allowed in test files
+    # Use mypy for static type checking.
+-   repo: https://github.com/pre-commit/mirrors-mypy
+    rev: 'v0.991'
+    hooks:
+      - id: mypy
+        files: >
+          (?x)^(
+            src/**/.*\.py|
```

Comment:
This will be a big behavior change for developers and we haven't proposed it to the team. Let's only do it for our pandas code for now.